File: easy/to_lower_case/to_lower_case.py
class Solution(object):
    def toLowerCase(self, str):
        """
        :type str: str
        :rtype: str
        """
        for c in str:
            if (c >= 'A') and (c <= 'Z'):
                str = str.replace(c, chr(ord(c) + 32), 1)
        return str


s = Solution()
print("Solution 1 :", s.toLowerCase("Hello"))
print("Solution 2 :", s.toLowerCase("here"))
print("Solution 3 :", s.toLowerCase("LOVELY"))

File: easy/array/to_lower_case/to_lower_case.go
package main

import (
	"fmt"
	"strings"
)

func toLowerCase(str string) string {
	for _, c := range str {
		if (c >= 'A') && (c <= 'Z') {
			str = strings.Replace(str, string(c), string(c+32), 1)
		}
	}
	return str
}

func main() {
	fmt.Println("Solution 1 : ", toLowerCase("Hello"))
	fmt.Println("Solution 2 : ", toLowerCase("here"))
	fmt.Println("Solution 3 : ", toLowerCase("LOVELY"))
}
